third_party/move/move-compiler-v2/tests/reference-safety/bug_13973_branch_cond_borrowed.move

```diff
@@ -0,0 +1,10 @@
+module 0xCAFE::Module0 {
+    public fun f1(x: bool) {
+        let y: &mut bool =  &mut x;
+        if (x) {
```

Comment:
Might be good to add a test case with a while loop (branch condition being the mutably borrowed local) as well.